conwayste/src/client.rs

```diff
@@ -819,6 +824,11 @@ impl EventHandler for MainState {
             _ => {}
         }
 
+        if !do_not_quit {
```

Comment:
The 'not' in the name, combined with its Boolean-ness and the fact that it's being negated, is just all so confusing. I was probably the originator :man_facepalming:. Could you change it to something like `let mut quit = false`?

Reply:
Fixed, thanks!